Purchases/Purchasing/RCReceiptFetcher.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  RCReceiptFetcher.swift
+//  Purchases
+//
+//  Created by Javier de Martín Gil on 8/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc open class RCReceiptFetcher: NSObject {
+
+    @objc open func receiptData() -> Data? {
```

Comment:
Once the migration is finished, we will no longer need to keep this class `open`, since it's only meant to be accessed within this module. 
could you add a `TODO` here to serve as a reminder that we should make this class `internal` once the migration is finished?